### DISCUSSION TITLE
Optional IIIF compatibility / support

### DISCUSSION THREAD

**Discussion by pvetch** [wagtail/wagtail#9016] [Ideas]
Flagging this here to see if there is any desire for this amongst current Wagtail users.

IIIF = International Image Interoperability Framework (http://iiif.io/)
and it's a rapidly emerging standard gaining real traction in the cultural heritage and HEI sectors.  There is some complexity involved in that using it requires deploying an image server like IIPimage - http://iipimage.sourceforge.net/ - so this wouldn't be a trivial thing to implement.  However for certain sectors this would make Wagtail a really compelling option (to my knowledge the only CMS with any possible support for IIIF is Drupal but only in the context of the Islandora framework, which is very top-heavy).

For implementation specifics see:
http://iiif.io/technical-details.html

It's notable that our URL syntax for sharing images isn't a million miles away from the IIIF Image Request URL syntax.

**Comment by jonnyscholes** [comment]
FWIF I would be very excited to see this land in Wagtail. Between this and the focal points this makes Wagtail very attractive to perfectionist frontend developers and (client) content producers.

**Comment by thibaudcolas** [comment]
I’m going to move this to a Discussion as I think these days we’d generally recommend endeavours like this are done as third-party packages first, with any needed core change tracked separately.
